src/subdomains/core/aml/enums/aml-error.enum.ts

```diff
@@ -53,6 +53,7 @@ export enum AmlError {
   VIDEO_IDENT_MISSING = 'VideoIdentMissing',
   LIQUIDITY_LIMIT_EXCEEDED = 'LiquidityLimitExceeded',
   IBAN_CURRENCY_MISMATCH = 'IbanCurrencyMismatch',
+  MERGE_MAIL_PENDING = 'MergeMailPending',
 }
```

Comment:
Call it `MERGE_PENDING` or `ACCOUNT_MERGE_PENDING?`? Mail is just the type of notification we use to notify the user.